### PR TITLE
[HttpKernel] add missing class alias to satisfy autoloading

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 trigger_deprecation('symfony/http-kernel', '6.4', 'The "%s" class is deprecated, use "%s" instead.', FileLinkFormatter::class, ErrorHandlerFileLinkFormatter::class);
 
 class_exists(ErrorHandlerFileLinkFormatter::class);
+class_alias(ErrorHandlerFileLinkFormatter::class, FileLinkFormatter::class);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php]: the alias already exists in `ErrorHandlerFileLinkFormatter` so this should not be needed
are you checking cross version failures ? I removed the alias from 7.0, maybe that's the issue?

Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php]: tests are failing on 6.3 with 6.4 of the HttpKernel component installed: https://github.com/symfony/symfony/actions/runs/6550029688/job/17788242726#step:8:10353

Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php]: okay, but the ErrorHandler component is installed in version 7

Comment (nicolas-grekas) [reply src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php]: okay! then yes, we can add this alias! maybe wrap it in a class_exists(, false) check to not collide with the alias defined in the new class on 6.4?

Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php]: good point, I wasn't aware of the other class alias, `class_exists()` check added

**Pull request merged**